--- config.rb.orig	2022-12-12 13:15:56.680304755 -0500
+++ config.rb	2022-12-12 13:21:05.136082581 -0500
@@ -52,18 +52,18 @@
 ## command is running. When running as a daemon/service, this is put into a
 ## file in logs/archivesspace.out. You can change this file by changing the log
 ## value to a filepath that archivesspace has write access to.
-#AppConfig[:frontend_log] = "default"
+AppConfig[:frontend_log] = "/archivesspace/logs/archivesspace.out"
 ## Log level for the frontend, values: (everything) debug, info, warn, error, fatal (severe only)
-#AppConfig[:frontend_log_level] = "debug"
+AppConfig[:frontend_log_level] = "warn"
 ## Log level for the backend, values: (everything) debug, info, warn, error, fatal (severe only)
-#AppConfig[:backend_log] = "default"
-#AppConfig[:backend_log_level] = "debug"
+AppConfig[:backend_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:backend_log_level] = "warn"
 #
-#AppConfig[:pui_log] = "default"
-#AppConfig[:pui_log_level] = "debug"
+AppConfig[:pui_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:pui_log_level] = "info"
 #
-#AppConfig[:indexer_log] = "default"
-#AppConfig[:indexer_log_level] = "debug"
+AppConfig[:indexer_log] = "/archivesspace/logs/archivesspace.out"
+AppConfig[:indexer_log_level] = "info"
 #
 #
 ## Set to true to log all SQL statements.  Note that this will have a performance
@@ -159,7 +159,6 @@
 #AppConfig[:solr_facet_limit] = 100
 #
 #AppConfig[:default_page_size] = 10
-#AppConfig[:max_boolean_queries] = 1024 # ArchivesSpace Solr default
 #AppConfig[:max_page_size] = 250
 #
 ## An option to change the length of the abstracts on the collections overview page
@@ -292,7 +291,7 @@
 #
 ## option to enable custom reports
 ## USE WITH CAUTION - running custom reports that are too complex may cause ASpace to crash
-#AppConfig[:enable_custom_reports] = false
+AppConfig[:enable_custom_reports] = true
 #
 ## Path to system Java -- required when creating PDFs on Windows
 #AppConfig[:path_to_java] = "java"
